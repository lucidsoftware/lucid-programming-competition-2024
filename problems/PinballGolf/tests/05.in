version https://git-lfs.github.com/spec/v1
oid sha256:8a53198374b7521dbffe8a16c3aea8ce967b2c51aebd1b354800ab22453bc9d0
size 2173
