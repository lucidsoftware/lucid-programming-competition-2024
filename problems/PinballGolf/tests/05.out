0.15048168
